core/src/test/scala/org/apache/spark/sql/BaseTiSparkTest.scala

```diff
@@ -377,6 +384,18 @@ class BaseTiSparkTest extends QueryTest with SharedSQLContext {
       }
     }
 
+    // test tiflash's result
```

Comment:
when `TiFlash` is tested, you should disable all other tests, because we only want to test if `TiFlash` is correct, regardless of TiSpark.

Reply:
TiFlash test is just an if branch, it does not affect other tests.

Reply:
I mean that when you try to test TiFlash, you should disable tests concerning TiSpark. Or else the result may not be correct because we only need one of the three tests to succeed in order to mark the test as passed, while in fact we need TiFlash test to pass strictly.

Reply:
understood that a fail() is called when comparison fails.